pyani_plus/public_cli_args.py

```diff
@@ -286,3 +286,11 @@ def version_callback(*, version: bool = False) -> None:
         rich_help_panel="Method parameters",
     ),
 ]
+OPT_ARG_TYPE_HEATMAP_ANNOTATED_MODE = Annotated[
+    bool,
+    typer.Option(
+        "--annotate",
+        help="Add annotations to ANI heatmaps.",
+        rich_help_panel="Plot-Run parameters",
```

Comment:
A whole panel for one option is premature (although `--label` might fit, but that's used in other contexts too like export run).